test/dom/dom-implementation.test.js

```diff
@@ -0,0 +1,165 @@
+'use strict'
+
+const {
+	DocumentType,
+	DOMImplementation,
+	Element,
+	Node,
+	NodeList,
+} = require('../../lib/dom')
```

Comment:
I would favor that we export and use these from `lib/index.js`, as I proposed in PR #233.

Reply:
Does it mean you want to land 233 first?